### DISCUSSION TITLE
Split window then GotoFile not working

### DISCUSSION THREAD

**Discussion by GGalizzi** [JetBrains/ideavim#501] [Q&A]
I want keybindings that split the window and open the `GotoFile` modal. i.e Open a given file in a split with one keybind.

e.g

```
nmap <leader>l <Action>(SplitVertically) <Action>(GotoFile)
nmap <leader>j :split<CR> <Action>(GotoFile)
```

Problem: The above does not work, the splits happen, but the GotoFile never opens.

I've also tried switching the order of operations

```
nmap <leader>l <Action>(GotoFile) <Action>(SplitVertically) 
```

This does open the GotoFile window and splits vertically, but the focus goes to the editor.

Ideas?

**Comment by AlexPl292** [comment]
Hi, please follow this ticket for the updates: https://youtrack.jetbrains.com/issue/VIM-2065/Executing-two-actions-via-one-mapping-doesn-t-behave-as-expected
